valhalla/jawn/src/controllers/public/requestController.ts

```diff
@@ -58,7 +60,9 @@ export class RequestController extends Controller {
    *  "offset": 0,
    *  "sort": {
    *    "created_at": "desc"
-   *  }
+   *  },
```

Comment:
nit: can we rmv this comment?

Reply:
we need this for example in swagger 
![image](https://github.com/Helicone/helicone/assets/38397684/1f0036f7-ddc8-410d-aee3-154e56a9e3c8)